tinker/templates/admin/redirects/home.html

```diff
@@ -80,12 +80,44 @@
                 $(".delete-column").on('click', function () {
                     $("#modal-confirm").attr("data-delete-id", $(this).data('id'));
                 });
-                //Listens for a click to get info on a redirect
+
+                // Listens for a click to get info on a redirect
                 $(".info-column").on('click',function(){
-                    $("#modal-id-display").text($(this).data('id'));
-                    $("#modal-username-display").text($(this).data("username"));
-                    $("#modal-date-display").text($(this).data("timestamp"));
-                    $("#modal-exp-date-display").text($(this).data("exp-date"));
+                    // Displaying data
+                    $("#modal-id-display").text($(this).data('id')).show();
```

Comment:
this block is quite large. Thoughts on simplifying it at all?

Reply:
It is big... I don't know how we would simplify it off the top of my head though because of the way we chose to hide and show the edit fields. Each individual field at this point needs to be either hidden or shown. I could pull it out to a method, but this is the only appearance of this code, and it's the only thing happening when the "info" button gets clicked.

Reply:
I believe you could add a class to act as a "grouping". This allows you to show/hide all of them at once. Setting the data can kind of be simplified by creating a dict with key/values as id/data. There is a chance this will be more complicated, but it might be worth a try.

Reply:
I was able to figure  out how to show and hide things as a group. It only made this spot a little simpler, but I did the same thing on the section that is a little lower and it made it significantly simpler. I don't think setting the data in a dictionary would help simplify things much. It will be about the same number of lines of code, and then I'd probably have to add a method that sets the values to display.

Reply:
Sounds good. I think its totally fine to avoid setting the data with dictionaries.